man/systemd.network.xml

```diff
@@ -2338,7 +2338,7 @@
           <term><varname>BitRate=</varname></term>
           <listitem>
             <para>The bitrate of CAN device in bits per second. The usual SI prefixes (K, M) with the base of 1000 can
-            be used here.</para>
+            be used here. Takes a number in the range 1-4294967296.</para>
```

Comment:
the upper end of the range is 2^32-1, not 2^32, no?

Also, we usually write ranges as "x…y" in prose, not "x-y" so far (latter usually meaning minus)